dasgo/resources.go

```diff
@@ -5,20 +5,22 @@ import (
 	"time"
 )
 
+// TODO: I assumed type snowflake is nonzero
```

Comment:
A snowflake with ID 0 will not be valid if it's included or not, so it is correct to assume it cannot be zero as a valid value.